src/RoslynInsertionTool/scripts/VS-Insertion.ps1

```diff
@@ -54,8 +54,8 @@ Do-Insertion -component "Live Unit Testing"  -queueName "TestImpact-Signed"
 ###
 
 #Do-Insertion -component "Roslyn"  -queueName "Roslyn-Signed"  -fromBranch "dev15.8.x-vs-deps"        -toBranch "rel/d15.8"  -insertToolset "false"  -insertDevdiv "false"  -updatecorextlibraries "true"  -queueValidation "true"
-Do-Insertion -component "Roslyn"  -queueName "Roslyn-Signed"  -fromBranch "dev15.9.x-vs-deps"        -toBranch "rel/d15.9"  -insertToolset "false"  -insertDevdiv "false"  -updatecorextlibraries "true"  -queueValidation "true"
-Do-Insertion -component "Roslyn"  -queueName "Roslyn-Signed"  -fromBranch "dev16.0-vs-deps"          -toBranch "rel/d16.0"  -insertToolset "false"  -insertDevdiv "false"  -updatecorextlibraries "true"  -queueValidation "true"
+#Do-Insertion -component "Roslyn"  -queueName "Roslyn-Signed"  -fromBranch "dev15.9.x-vs-deps"        -toBranch "rel/d15.9"  -insertToolset "false"  -insertDevdiv "false"  -updatecorextlibraries "true"  -queueValidation "true"
+#Do-Insertion -component "Roslyn"  -queueName "Roslyn-Signed"  -fromBranch "dev16.0-vs-deps"          -toBranch "rel/d16.0"  -insertToolset "false"  -insertDevdiv "false"  -updatecorextlibraries "true"  -queueValidation "true"
 Do-Insertion -component "Roslyn"  -queueName "Roslyn-Signed"  -fromBranch "release/dev16.3-vs-deps"  -toBranch "lab/d16.3stg"  -insertToolset "false"  -insertDevdiv "false"  -updatecorextlibraries "true"  -queueValidation "true"
```

Comment:
There is no lab/d16.3stg branch. I believe this should be rel/d16.3 after the snap. Also, we have an automatic release pipeline that I could re-enable https://devdiv.visualstudio.com/DevDiv/_release?view=mine&definitionId=1838